packages/data-point/lib/reducer-list/index.js

```diff
@@ -0,0 +1,12 @@
+/* eslint global-require: 0 */
```

Comment:
is this new code or should had been a git mv?

Reply:
should have been a git mv